admin/proc-rhdh-deploy-aks.adoc

```diff
@@ -0,0 +1,315 @@
+[id='proc-rhdh-deploy-aks_{context}']
+= Deploying {product} on Azure Kubernetes Services (AKS)
+
+You can deploy your {product-short} application on Azure Kubernetes Services (AKS) to access a comprehensive solution for building, testing, and deploying applications.
+
+.Prerequisites
+* You have an Azure account with active subscription.
+* You have installed Azure CLI in your machine and configured the Resource Group and Cluster. For more information, see https://learn.microsoft.com/en-us/cli/azure/install-azure-cli[How to install the Azure CLI]. 
++
+--
+You can perform the following steps to configure the Resource Group and Cluster:
```

Comment:
These steps specifically detail the configuration of the custom resource and cluster using the terminal interface rather than any UI. By including these steps alongside their prerequisites, users can complete the necessary configurations without being redirected to other resources for minor setups (Which are specifically not documented, hence not duplicated). This approach aims to enhance user-friendliness.